Form/Type/DocumentType.php

```diff
@@ -66,16 +66,22 @@ public function setDefaultOptions(OptionsResolverInterface $resolver)
             return $registry->getManager($manager);
         };
 
-        $resolver->setNormalizers(array(
-            'em' => $normalizer,
-        ));
+        $resolver->setNormalizer('em', $normalizer);
```

Comment:
You then have to bump the required version of the OptionsResolver component to `2.6` or higher.